library/std/src/os/rtems/mod.rs

```diff
@@ -0,0 +1,5 @@
+#![stable(feature = "raw_ext", since = "1.1.0")]
+#![forbid(unsafe_op_in_unsafe_fn)]
+
+pub mod fs;
+mod raw;
```

Comment:
@tgross35 Sorry, I have to open this again. I compiled only for the host architecture locally. This works, because this file is ignored.
Now, I also tried to compile for the target and I get the import errors again:

```
error[E0432]: unresolved import `super::platform::raw`
  --> std/src/os/unix/raw.rs:27:26
   |
27 | pub use super::platform::raw::pthread_t;
   |                          ^^^ could not find `raw` in `platform`

error[E0432]: unresolved import `super::platform::raw`
  --> std/src/os/unix/raw.rs:30:26
   |
30 | pub use super::platform::raw::{blkcnt_t, time_t};
   |                          ^^^ could not find `raw` in `platform`

error[E0432]: unresolved import `super::platform::raw`
  --> std/src/os/unix/raw.rs:33:26
   |
33 | pub use super::platform::raw::{blksize_t, dev_t, ino_t, mode_t, nlink_t, off_t};
   |                          ^^^ could not find `raw` in `platform`
```

I also tried some other things with `pub(crate)` and similar, but in the end I am not sure if there is an easy solution. The file `os/unix/raw.rs` imports the platform's `raw` module. Doesn't that mean I need to declare it public in `rtems/mod.rs` in order to have it visible in `unix/raw.rs`?

Reply:
What happened with `pub(crate)`? It seems like that should work.

Reply:
Thanks. I didn't know that this can also be applied to a whole module.